usdt/Cargo.toml

```diff
@@ -9,11 +9,11 @@ description = "Dust your Rust with USDT probes"
 repository = "https://github.com/oxidecomputer/usdt.git"
 
 [dependencies]
-dtrace-parser = { path = "../dtrace-parser", optional = true }
+dtrace-parser = { path = "../dtrace-parser", optional = true, version = "0.1.14" }
 serde = "1"
-usdt-impl = { path = "../usdt-impl", default-features = false }
-usdt-macro = { path = "../usdt-macro", default-features = false }
-usdt-attr-macro = { path = "../usdt-attr-macro", default-features = false }
+usdt-impl = { path = "../usdt-impl", default-features = false, version = "0.3.4" }
+usdt-macro = { path = "../usdt-macro", default-features = false, version = "0.3.4" }
```

Comment:
One more thought: Should the version dependencies here be exact (i.e. `= 0.3.4` instead of just `0.3.4`) so that floating to a new patch version is not allowed? (This was what bit me with the Propolis change--pinning back to usdt 0.3.2 wasn't sufficient to keep it from grabbing usdt-macro 0.3.3 and getting the incorrect usdt-impl dependency.)

Reply:
Ah, nice catch, thank you. Incoming!

Reply:
Done in 1d10020